packages/@glimmer/compiler/lib/javascript-compiler.ts

```diff
@@ -243,8 +241,11 @@ export default class JavaScriptCompiler
   modifier(name: string) {
     let params = this.popValue<Params>();
     let hash = this.popValue<Hash>();
-
-    this.push([SexpOpcodes.Modifier, name, params, hash]);
+    if (this.currentComponent) {
+      this.currentComponent.push([SexpOpcodes.Modifier, name, params, hash]);
+    } else {
+      this.push([SexpOpcodes.Modifier, name, params, hash]);
+    }
```

Comment:
This shouldn't be necessary: `this.push` goes through `this.currentBlock.push`, which should be the same as `this.currentComponent`. However, I think [this list](https://github.com/glimmerjs/glimmer-vm/blob/master/packages/%40glimmer/compiler/lib/javascript-compiler.ts#L91-L101) need to be updated to account for modifiers (push it into `this.attributes`).

Reply:
Reverted to a regular push